docs/linux/zbd-api.md

```diff
@@ -193,7 +178,7 @@ the enumeration `enum blk_zone_type`.
  * @BLK_ZONE_TYPE_SEQWRITE_REQ: The zone must be written sequentially
  * @BLK_ZONE_TYPE_SEQWRITE_PREF: The zone can be written non-sequentially
  *
- * Any other value not defined is reserved and must be considered as invalid.
+ * Any other value not defined is reserved and must be considered invalid.
```

Comment:
We should not change this as it is a copy-paste of the kernel header file... I will fix the typo there.
I will this up when applying.